.travis.yml

```diff
@@ -28,6 +28,9 @@ install:
         - wget https://github.com/libcheck/check/releases/download/0.12.0/check-0.12.0.tar.gz
         - tar -xvf check-0.12.0.tar.gz
         - pushd check-0.12.0 && ./configure --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu && make -j48 && sudo make install && popd
+        - wget https://s3.amazonaws.com/json-c_releases/releases/json-c-0.13.1.tar.gz
```

Comment:
Why do you download json-c from aws when there's a github repo with releases?

https://github.com/json-c/json-c/
https://github.com/json-c/json-c/archive/json-c-0.13.1-20180305.tar.gz

Reply:
Done!